test/common.py

```diff
@@ -1983,7 +1983,7 @@ def _build_and_run(self, filename, expected_output, args=None,
       # TODO once standalone wasm support is more stable, apply use_all_engines
       # like with js engines, but for now as we bring it up, test in all of them
       if not self.wasm_engines:
-        logger.warning('no wasm engine was found to run the standalone part of this test')
+        self.skipTest('no wasm engine was found to run the standalone part of this test')
```

Comment:
It might be safer to do what we do for various features like this:

https://github.com/emscripten-core/emscripten/blob/ca3d456e4e7c861dc632e475081400a5c79a4204/test/common.py#L1053-L1056

We error by default, so we can't forget to set up all the test requirements, and we allow manual opt-out instead of opt-in.

Reply:
Yes, something like `EMTEST_SKIP_WASM_ENGINE` probably makes the most sense.